### ISSUE TITLE
Inaccessible xml schema

### ISSUE THREAD

**Issue by ralf0131** [apache/incubator-dubbo#1629]
In the demo code we have dubbo-demo-provider.xml

```xml
<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xmlns:dubbo="http://code.alibabatech.com/schema/dubbo"
       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://code.alibabatech.com/schema/dubbo http://code.alibabatech.com/schema/dubbo/dubbo.xsd">
    <dubbo:application name="demo-provider"/>
    <dubbo:registry address="multicast://224.5.6.7:1234"/>
    <dubbo:protocol name="dubbo" port="20880"/>
    <dubbo:service interface="com.alibaba.dubbo.demo.DemoService" ref="demoService"/>
    <bean id="demoService" class="com.alibaba.dubbo.demo.provider.DemoServiceImpl"/>
</beans>
```

The schema location `http://code.alibabatech.com/schema/dubbo/dubbo.xsd` is not accessible currently. It is not a problem using Intellij IDEA since it will find it automatically in the packaged jar file. However if we are using Eclipse, the following error message will occur:

```
cvc-complex-type.2.4.c: The matching wildcard is strict, but no declaration can be found for element 
 'dubbo:application'.
```

Proposal:
Host dubbo.xsd under dubbo.apache.org, that means changing the namespace from `http://code.alibabatech.com/schema/dubbo` to `http://dubbo.apache.org/schema/dubb`

Other things to consider:

How to keep dubbo.xsd update to date is a problem, one way I can think is when deploying codes to dubbo.apache.org, download the latest dubbo.xsd in the deploy script.

**Comment by ralf0131** [comment]
This issue has been fixed from 2.6.2 onwards.

**Comment by carrymumu** [comment]
**http://dubbo.apache.org/schema/dubbo** is better....
